charts/fluent-operator/Chart.yaml

```diff
@@ -15,9 +15,9 @@ description: A Helm chart for Kubernetes
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 1.5.2
+version: 1.5.3
```

Comment:
Let make chart version the same as appversion

Reply:
fixed